install_config/aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,254 @@
+
+----
+$ oc edit configmap logging-deployer
+  [change es-cluster-size value to 5]
+
+$ oc new-app logging-deployer-template --param MODE=reinstall
+----
+
+[[install-config-aggregate-logging-sizing-guidelines-storage]]
+== Storage Considerations 
+An Elasticsearch index is a collection of shards and its corresponding replica shards. This is how ES implements high availability internally, therefore there is little need to use hardware based mirroring RAID variants. RAID 0 can still be used to increase overall disk performance. +
+Every search request needs to hit a copy of every shard in the index.
+Each ES instance requires its own individual storage, but an {product-title} deployment can only provide volumes shared by all of its pods, which again means that Elasticsearch shouldn't be implemented with a single node. +
+
+A xref:../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[persistent volume] should be added to each Elasticsearch deployment configuration so that we have one volume per link:https://www.elastic.co/guide/en/elasticsearch/guide/current/replica-shards.html[replica shard]. On {product-title} this is often achieved through xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[Persistent Volume Claims] +
+
+* 1 volume per shard
+
+* 1 volume per replica shard
+  
+The PVCs must be named based on the `es-pvc-prefix` setting. Refer to xref:../install_config/aggregate_logging.adoc#overview[Persistent Elasticsearch Storage] for more details. Below are capacity planning guidelines for {product-title} Aggregated Logging.
+
+*Example scenario*
+
+Assumptions: +
+1) Which app: Apache +
+2) Bytes per line: 256 +
+3) Lines per second load on app: 1 +
+4) Raw text data -> json +
+
+
+Baseline (256 characters per minute -> 15KB/min)
+
+[cols="3,4",options="header"]
+|===
+|Logging infra pods
+|Storage throughput
+
+|3 es +
+1 kibana +
+1 curator +
+1 fluentd +
+| 6 pods total: 90000 x 86400 = 7,7 GB/day
+
+|3 es +
+1 kibana +
+1 curator +
+11 fluentd 
+| 16 pods total: 225000 x 86400 = 24,0 GB/day
+
+|3 es +
+1 kibana +
+1 curator +
+20 fluentd 
+|25 pods total: 225000 x 86400 = 32,4 GB/day
+|===
+
+
+Calculating total logging throughput and disk space required for your logging environment requires knowledge of your application. For example, if one of your applications on average logs 10 lines-per-second, each 256 bytes-per-line, calculate per-application throughput and disk space as follows:
+
+----
+ (bytes-per-line * (lines-per-second) = 2560 bytes per app per second
+ (2560) * (number-of-pods-per-node,100) = 256,000 bytes per second per node
+ 256k * (number-of-nodes) = total logging throughput per cluster
+----
+
+Fluentd ships any logs from `/var/log/messages` and `/var/lib/docker/containers/*` to Elasticsearch. +
+More details xref:../install_config/install/prerequisites.adoc#managing-docker-container-logs[here].
```

Comment:
Reword this directive. Perhaps:
xref:../install_config/install/prerequisites.adoc#managing-docker-container-log[Learn more].